src/sidebars/sidebar.html

```diff
@@ -17,8 +17,26 @@
             </div>
         </div>
     </div>
-    <div class="search {{sidebarCtrl.searchClass()}}" ng-controller="SearchCtrl as searchCtrl" ng-click="sidebarCtrl.searchClick()">
-        <div class="sidetab-top">
```

Comment:
In that case given it's now specific to the favourites section it could probably do with a rename.